seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -4779,7 +4792,7 @@
     },
     {
       "id": "56533eb9ac21ba0edf2244cc",
-      "title": "Accessing Nested Objects",
+      "title": "Access Nested Objects",
```

Comment:
I don't remember doing this, maybe it crept in over cycles of rebasing, thanks for catching, I will update.